magic_8_ball_bot.py

```diff
@@ -1,27 +1,31 @@
-#Imports required modules
+# Imports required modules
 import discord
 from discord.ext import commands
 import random
 
-#Dictionary pairing an integer with a value that will be returned 
-mag_dic = {1:'It is certain', 2:'It is decidedly so', 3:'Without a doubt', 4:'Yes definitely', 5:'You may rely on it',
+# Dictionary pairing an integer with a value that will be returned 
+MAG_DICT = {1:'It is certain', 2:'It is decidedly so', 3:'Without a doubt', 4:'Yes definitely', 5:'You may rely on it',

```

Comment:
FYI: We can just imply that this is a constant set by making the name be ALL CAPS. See [PEP8 Constants](https://peps.python.org/pep-0008/#constants).